integration-libs/opf/base/core/connectors/otp.connector.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { OtpAdapter } from './otp.adapter';
+
+@Injectable()
+export class OtpConnector {
```

Comment:
[MEDIUM]

1. The abbreviation OTP is not obvious. Please resolve it in the JSDoc of each method that contains this abbreviation.
2. The name of this Connector is too generic IMHO. One-time-passwords can exist in various domains like payment or authenticateion. So to be more precise, I'd suggest to prefix it with `Opf` (`OpfOtpConnector`), like we do for `OpfPaymentConnector` and `OpfOrderConnector`.

Reply:
ATM we have access code service in a cart feature lib.